src/nvim/CMakeLists.txt

```diff
@@ -183,7 +189,7 @@ foreach(sfile ${NEOVIM_SOURCES}
     DEPENDS "${HEADER_GENERATOR}" "${sfile}"
     )
   list(APPEND NEOVIM_GENERATED_SOURCES "${gf1}")
-  list(APPEND NEOVIM_GENERATED_SOURCES "${gf2}")
```

Comment:
And, I would like to change the variable names of gf1, gf2, gf3 to gf_c_h, gf_h_h, gf_i to make it easier to be understood.